src/mono/wasm/build/WasmApp.targets

```diff
@@ -6,14 +6,35 @@
   <UsingTask TaskName="PInvokeTableGenerator" AssemblyFile="$(WasmAppBuilderTasksAssemblyPath)" />
 
   <!--
-      Public properties:
-      - WasmBuildDir    - Directory where build artifacts are stored.
-      - WasmBuildNative - Whenever to build the native executable. Defaults to false.
-      - WasmNativeStrip - Whenever to strip the native executable. Defaults to true.
+      Public properties (required):
+      - $(WasmBuildDir)    - Directory where build artifacts are stored (required if WasmBuildNative==true)
+      - $(WasmAppDir)      - AppBundle dir
+      - $(WasmMainAssemblyPath)
+      - $(WasmMainJSPath)
+      - $(EMSDK_PATH)      - points to the emscripten sdk location.
+
+      Public properties (optional):
+      - $(WasmBuildNative)   - Whenever to build the native executable. Defaults to false.
+      - $(WasmNativeStrip)   - Whenever to strip the native executable. Defaults to true.
+      - $(RunAOTCompilation) - Defaults to false.
+      - $(EnableProfiler)    - Defaults to false.
+      - $(WasmDebugLevel)
```

Comment:
It would be simpler to just have WasmProfilers, and set EnableProfilers if its non-empty.